src/main.rs

```diff
@@ -248,33 +250,33 @@ fn main() {
         .author(crate_authors!())
         .version(crate_version!())
         .arg_required_else_help(true)
-        .arg(
```

Comment:
Args were reordered in previous commit from @cgzones, would prefer to keep alphabetical order.